CHANGELOG.md

```
@@ -2,6 +2,22 @@
 
 ## [Unreleased]
 
+## [0.9.5] - 2026-05-21
+
+### New
```

markdownlint-cli2 (0.22.1)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)